driver-tests/osgi/pom.xml

```diff
@@ -203,6 +197,16 @@
                     <groups>${test.groups}</groups>
                     <systemPropertyVariables>
                         <cassandra.version>${cassandra.version}</cassandra.version>
+                        <!-- pull in declared version properties, this ensures we test with the same versions used by the driver -->
+                        <guava.version>${guava.version}</guava.version>
```

Comment:
This is very nice.  Wish I had thought of it from the start.  Much better than it being hardcoded in the source.